doc/datavolumes.md

```diff
@@ -1,7 +1,7 @@
 # Data Volumes
 
 ## Introduction
-Data Volumes(DV) are an abstraction on top of Persistent Volume Claims(PVC) and the Containerized Data Importer(CDI). The DV will monitor and orchestrate the upload/import of the data into the PVC. Once the process is completed, the DV will be in a consistent state that allow consumers to make certain assumptions about the DV in order to progress their own orchestration.
+Data Volumes(DV) are an abstraction on top of Persistent Volume Claims(PVC) and the Containerized Data Importer(CDI). The DV will monitor and orchestrate the upload/import of the data into the PVC. Specific [DV annotations](datavolume-annotation.md) are passed to the transfer pods to control their behavior. Once the process is completed, the DV will be in a consistent state that allow consumers to make certain assumptions about the DV in order to progress their own orchestration.
```

Comment:
I would drop this from the introduction, wanting to pass annotations is a niche use case that wouldn't interest everyone, and we're putting more of a burden to understand what datavolumes are

Reply:
moved down